websites/M/MangaKatana/presence.ts

```diff
@@ -0,0 +1,182 @@
+const presence = new Presence({
+		clientId: "1104842127034306724",
+	}),
+	browsingTimestamp = Math.floor(Date.now() / 1000),
+	{ pathname, href, search } = document.location,
+	path = pathname.split("/"),
+	title = document
+		.querySelector("meta[property='og:title']")
+		.getAttribute("content");
+
+async function getStrings() {
+	return presence.getStrings(
+		{
+			editing: "general.editing",
+			genre: "general.genre",
+			manga: "general.manga",
+			reading: "general.reading",
+			readingAbout: "general.readingAbout",
+			search: "general.search",
+			searchFor: "general.searchFor",
+			searchSomething: "general.searchSomething",
+			viewing: "general.viewing",
+			viewHome: "general.viewHome",
+			viewPage: "general.viewPage",
+			buttonViewPage: "general.buttonViewPage",
+		},
+		await presence.getSetting<string>("lang").catch(() => "en")
+	);
+}
+
+const enum Assets {
+	Logo = "https://i.imgur.com/41vZ7bA.png",
+}
+
+let strings: Awaited<ReturnType<typeof getStrings>>,
+	oldLang: string = null;
+
+function textContent(tags: string) {
+	return document.querySelector(tags)?.textContent?.trim();
+}
+
+const observer = new IntersectionObserver(
+	(entries, observer) => {
+		for (const entry of entries) {
+			if (entry.isIntersecting) {
+				observer.unobserve(entry.target);
+				localStorage.setItem(
+					"currentMangaPage",
+					(<HTMLImageElement>entry.target).src
+				);
```

Comment:
You are [not allowed](https://github.com/PreMiD/Presences/blob/main/.github/CONTRIBUTING.md#presencets) to access `localStorage`. Is there a reason you need to use `localStorage` over a local variable?